api/docker/php/docker-entrypoint.sh

```diff
@@ -19,11 +19,6 @@ if [ "$1" = 'php-fpm' ] || [ "$1" = 'php' ] || [ "$1" = 'bin/console' ]; then
 
 	if [ "$APP_ENV" != 'prod' ]; then
 		composer install --prefer-dist --no-progress --no-interaction
-
-		echo "Making sure public / private keys for JWT exist..."
-		php bin/console lexik:jwt:generate-keypair --skip-if-exists --no-interaction
```

Comment:
does keycloak generate them automatically?

Reply:
Yes, Keycloak provides its own certificate files, there is no need to generate them from the Symfony project